Makefile

```diff
@@ -20,6 +20,9 @@ build-windows-amd:
 	GOOS=windows GOARCH=amd64 go build -tags $(or $(datree_build_env),staging) -ldflags="-X github.com/datreeio/datree/cmd.CliVersion=1.0.0"
 
 test:
+	# when changing rule logic and/or tests, we want to ensure our new logic is used when running "make test"
+	# if we don't delete `defaultRules.yaml`, the tests will use the old logic
+	rm -f ~/.datree/defaultRules.yaml
```

Comment:
does it work without `-f`? probably better if possible

Reply:
`-f` is needed - if the file is already deleted, the `rm` command will fail and exit unless `-f` is used